### PR TITLE
Account for ground truth in progress, export and when adding answers

### PR DESCRIPTION
**Pull request by MikeOverkamp-diag**

### CODE REVIEW

**Comment by jmsmkn** [comment]
I saw this test fail earlier today on master, but re-running the action solved it. I don't recall it being flaky before.

File: app/grandchallenge/reader_studies/templatetags/get_ground_truth.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from django import template
+
+register = template.Library()
+
+
+@register.simple_tag
+def get_ground_truth(obj, image, question):
+    """Get the auth token for the user."""
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/templatetags/get_ground_truth.py]: Update the docstring

File: app/grandchallenge/reader_studies/models.py [hunk 2]
```diff
@@ -339,39 +343,47 @@ def get_progress_for_user(self, user):
...
-                        readerstudies=self, answers__isnull=True
+        # Group the answers by images and filter out the images that
+        # have an inadequate amount of answers
+        unanswered_images = (
+            answers.order_by("images__name")
+            .values("images__name")
+            .annotate(answer_count=Count("images__name"))
+            .filter(answer_count__lt=self.answerable_question_count)
+        )
+        image_names = set(
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/models.py]: Wow! Nothing to change here but I think that we probably need a model manager at this point.

File: app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html [hunk 3]
```diff
@@ -24,7 +25,9 @@ <h2 class="h3">Statistics per case</h2>
                     <th>Image name</th>
                     <th>Total score / max score</th>
                     <th>Average score</th>
-                    <th>Ground truth</th>
+                    {% for question in object.statistics.questions %}
```
Comment (jmsmkn) [inline app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html]: In `results_list.html` there is some js for hiding multiple columns at once, we will probably need that here in the future.

File: app/grandchallenge/reader_studies/templatetags/get_ground_truth.py (hunk 1 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/templatetags/get_ground_truth.py]: Changed it locally. I'll include it in the next PR (making a separate PR for would be a bit much I'd say).

File: app/grandchallenge/reader_studies/models.py (hunk 2 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/models.py]: Added an issue for this: https://github.com/comic/grand-challenge.org/issues/1154

File: app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html (hunk 3 above)
Comment (MikeOverkamp-diag) [reply app/grandchallenge/reader_studies/templates/reader_studies/readerstudy_statistics.html]: Added an issue for this as well: https://github.com/comic/grand-challenge.org/issues/1155

**Pull request merged**